plugins/vep/Hpo.pm

```diff
@@ -0,0 +1,77 @@
+package Hpo;
+
+use strict;
+use warnings;
+
+use Bio::EnsEMBL::Variation::Utils::BaseVepPlugin;
+use base qw(Bio::EnsEMBL::Variation::Utils::BaseVepPlugin);
+
+=head1 NAME
+ Hpo
+=head1 SYNOPSIS
+ mv Hpo.pm ~/.vep/Plugins
+ ./vep -i variations.vcf --plugin Hpo,/FULL_PATH_TO_GENES_TO_PHENOTYPE_FILE/genes_to_phenotype.txt,HP:0000275;HP:0000276
```

Comment:
```suggestion
 ./vep -i variations.vcf --plugin Hpo,/FULL_PATH_TO_GENES_TO_PHENOTYPE_FILE/genes_to_phenotype.txt,"HP:0000275;HP:0000276"
```
otherwise the ";" is the end of the statement